py/specter/psf/spotgrid.py

```diff
@@ -57,16 +58,37 @@ def _xypix(self, ispec, wavelength):
         Return xslice, yslice, pix for PSF at spectrum ispec, wavelength
         """
         return self._xypix_interp(ispec, wavelength)
+        
+    @jit(nopython=True,cache=True)
+    def new_pixshift(dx,dy,w00,w10,w01,w11,pix_spot_values,rebin):
+        """
+        Return resampled_pix_spot_values 
```

Comment:
Input and output documented. Now passing in xc and yc and calculating weighting factors (w00, w10...) and dx, dy within the function. Changed first instance of dx, dy into shiftx, shifty to avoid confusion.